gmp.spec

```diff
@@ -1,2 +1,2 @@
-### RPM external gmp 4.2.1
-Source: ftp://mirrors.kernel.org/gnu/%n/%n-%realversion.tar.gz
+### RPM external gmp 6.1.2
```

Comment:
It was part of the CMSDIST and I got a compiler error when I was building it. So I decided to update it and the build error went away.